vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_tests/application_code/nuvoton_code/entropy_hardware_poll.c

```diff
@@ -0,0 +1,221 @@
+/*
```

Comment:
Why do they have what appear to be two implementations of this? I.e. this seems to be a duplicate file. If it's not an exact duplicate, perhaps they could at least refactor the common parts.

Reply:
@dcgaws , I just followed existing other board's architecture. Other boards regard `entropy_hardware_poll.c` as user application part to let user DIY for furthermore TRNG implementation. I consider it could be regarded as weak function in common part and user could override it if user has better TRNG implementation.